0.8.0/_modules/torchaudio/functional/functional.html

```diff
@@ -1311,17 +1311,21 @@ <h1>Source code for torchaudio.functional.functional</h1><div class="highlight">
 <span class="p">)</span> <span class="o">-&gt;</span> <span class="n">Tensor</span><span class="p">:</span>
     <span class="sa">r</span><span class="sd">&quot;&quot;&quot;</span>
 <span class="sd">    Applies codecs as a form of augmentation</span>
+
 <span class="sd">    Args:</span>
 <span class="sd">        waveform (Tensor): Audio data. Must be 2 dimensional. See also ```channels_first```</span>
 <span class="sd">        sample_rate (int): Sample rate of the audio waveform</span>
 <span class="sd">        format (str): file format</span>
```

Comment:
I do not see the change made [here](https://github.com/pytorch/audio/pull/1317/files#diff-f69839488ce70c39d8c3db8c87790ac963364faf8aefcb38ff5b026ee9dca8c7R1019) in #1317 ?

Reply:
this file is for showing the source code. the one you are looking for are `0.8.0/transforms.html` `0.8.0/functional.html`

Reply:
Looking at 0.8.0/functional.html shows the outdated version too, see [comment](https://github.com/pytorch/audio/pull/1354#discussion_r587797575) below.

Reply:
oh you are right.